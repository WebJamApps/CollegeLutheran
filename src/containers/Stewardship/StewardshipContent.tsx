import ELCALogo from 'src/components/elcaLogo';
import parser from 'html-react-parser';
import { useContext } from 'react';
import { ContentContext } from 'src/providers/Content.provider';

const StewardshipContent = () => {
  const { content: { stewardshipPage } } = useContext(ContentContext);
  return (
    <div className="page-content">
      <div className="container-fluid">
        <p style={{ fontSize: '4pt', margin: '0' }}>&nbsp;</p>
        <div
          className="material-content elevation3"
          style={{
            maxWidth: '998px', margin: 'auto', minHeight: '74vh',
          }}
        >
          <h3 style={{ paddingTop: '22px', paddingBottom: '15px' }}>Stewardship</h3>
          <section style={{ marginTop: '20px', textAlign: 'left', marginBottom: '35px' }}>
            {parser(stewardshipPage && stewardshipPage.comments ? stewardshipPage.comments : '')}
          </section>
        </div>
        <ELCALogo />
      </div>
    </div>
  );
};
export default StewardshipContent;
