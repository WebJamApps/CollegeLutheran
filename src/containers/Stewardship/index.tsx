import DefaultContent from './StewardshipContent';
import commonUtils from '../../lib/commonUtils';

export function Stewardship(): JSX.Element {
  commonUtils.setTitleAndScroll('Stewardship', window.screen.width);
  return (<DefaultContent />);
}
