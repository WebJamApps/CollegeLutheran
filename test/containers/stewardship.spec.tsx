import { shallow } from 'enzyme';
import Stewardship from 'src/containers/Stewardship/index';
import DefaultContent from 'src/containers/Stewardship/StewardshipContent';

function setup() {
  const props = {};
  const wrapper = shallow(<Stewardship />);
  return { props, wrapper };
}

describe('Giving', () => {
  it('Renders the Giving component', () => {
    const { wrapper } = setup();
    expect(wrapper.find(DefaultContent)
      .dive()
      .find('div.page-content')
      .exists()).toBe(true);
  });
  it('renders snapshot correctly', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
