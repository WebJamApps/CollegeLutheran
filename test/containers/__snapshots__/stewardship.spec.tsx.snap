// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Giving renders snapshot correctly 1`] = `ShallowWrapper {}`;
